fixtures/signer-config.json

```diff
@@ -0,0 +1,16 @@
+{
+	"server": {
+		"http_addr": ":4444",
+		"grpc_addr": ":7899",
+		"tls_cert_file": "./notary-signer.crt",
+		"tls_key_file": "./notary-signer.key",
+		"client_ca_file": "./notary-server.crt"
+	},
+	"logging": {
+		"level": "debug"
+	},
+	"storage": {
+		"backend": "mysql",
+		"db_url": "dockercondemo:dockercondemo@tcp(notarymysql:3306)/dockercondemo"
```

Comment:
Would be good to talk to the registry guys to see how people use their images. Want to make sure nobody is going to spin it up in memory and expect that to be a healthy way to deploy production. I didn't comment because you've maintained the existing behaviour, but we may want to consider making the in memory deployment be explicit in the configuration rather than being the fallback/default when nothing is set.